python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -291,8 +294,8 @@ async def handle_summary_state(self) -> None:
         disconnect to the lower level components (or the mock_controller) when
         needed.
         """
-        self.log.info(f"handle_summary_state {self.summary_state}")
-        if self.disabled_or_enabled:
+        self.log.info(f"handle_summary_state {self.summary_state.name}")
+        if self.summary_state == salobj.State.ENABLED:
```

Comment:
You have a very good point and I reverted the change. It required a small modification to one of the unit test cases as well since the CSC will go to FAULT state already when going to DISABLED instead of ENABLED because the status commands time out as well as all other commands.